docs/docs/workflows/upgrade.md

```diff
@@ -58,6 +58,10 @@ Image and Kubernetes upgrades take longer.
 For each node in your cluster, a new node has to be created and joined.
 The process usually takes up to ten minutes per node.
 
+When applying an upgrade, backup files of the Terraform state and certain Custom Resources are created.
+The Terraform state backup can be used as a reference to the possible Terraform resource migration performed within an upgrade.
```

Comment:
```suggestion
The Terraform state backup can be used to restore previous resources in case an upgrade misconfigured or erroneously deleted a resource. 
```